front/app/modules/commercial/content_builder/admin/components/CraftComponents/Iframe/index.tsx

```diff
@@ -110,7 +116,7 @@ const IframeSettings = injectIntl(({ intl: { formatMessage } }) => {
             </span>
           }
           placeholder={formatMessage(messages.iframeHeightPlaceholder)}
-          type="text"
+          type="number"
           value={height}
```

Comment:
I notice that the placeholder shows "e.g. 120px" still. Do you think we should make this just "120"? Or maybe the "px" makes it clear that this number represents pixels?

Reply:
Maybe we could change the tooltip to "Height of the iframe window in pixels" and use just "e.g. 120" as a placeholder? Just an idea though.

Reply:
I think it should just be the number. This is coming in from Crowdin. Do you guys normally just change those there directly?

Reply:
Yep exactly :) Do you think we should update the copy in the "messages" file as well just to keep it consistent though?

Reply:
The one in the messages file is actually okay. I'll update the one in Crowdin now